src/api/public/apidocs-new/paths/search_channel_binary_id.yaml

```
@@ -0,0 +1,80 @@
+get:
+  summary: List channel objects that match a XPath condition.
+  description: |
+    Return a collection of channel objects that match a XPath condition.
+
+    This operation is the same as the one defined with `GET /search/channel/binary` with the exception of the results returned.
+    While the former operation returns a list of channel objects with release information, this one only returns a list of channel
+    objects with the channel definition related to the binary.
+    See example values of a succeeded request below.
+  security:
+    - basic_authentication: []
+  parameters:
+    - in: query
+      name: match
+      schema:
+        type: string
+      required: yes
+      description: |
+        Expression based in XPath.
+
+        Providing a value of `*` will return all channels.
```

```suggestion
        Not providing a value or providing a value of `*` will return all released binaries.
```
In this case it is allowed not providing a value.